bibdata.yml

```diff
@@ -1,20 +1,59 @@
 ---
-- hosts: 
-    - bibdata_staging
-    - bibdata_production
+- hosts: bibdata
   remote_user: pulsys
   become: true
   vars_files:
     - site_vars.yml
     - group_vars/bibdata/vault.yml
+  vars:
+    bibdata_db_host: localhost
+    passenger_server_name: bibdata-staging.*
+    samba_status: server
   roles:
     - role: pulibrary.ruby
     - role: pulibrary.deploy-user
     - role: pulibrary.passenger
     - role: pulibrary.redis
-    - {role: pulibrary.mariadb, when: bibdata_db_host == 'localhost'}
```

Comment:
assuming that we never install bibdata locally yes?

Reply:
This was there to install mariadb only staging machines. But it's no longer required because there is special config just for staging servers now.